version_control_standards.md

```diff
@@ -0,0 +1,39 @@
+# Version control standards
+
+These standards define how version control is applied to our code.
+
+Wherever possible these standards follow built-in versioning facilities for the languages,
+tools or frameworks we are using.
+
+## Rationale
+
+- Underpins a consistent, robust and repeatable release strategy
+- Ensures that we can have reliable dependency management between components even if they use different technologies
+
+## Standards
+
+### All code is held in a centrally managed Git repository
+
+The primary repository is in the Defra GitHub organisation.
+
+Azure DevOps Git or hosted GitLab may be used to provide mirrors of GitHub repositories.
+
+### Every repository has a protected master branch
```

Comment:
Just so you know as it surprises most people, protected doesn't prevent changes to master.

- it ensures you can't delete master
- it stops any forces pushes to master
- if you have them you can require things to pass CI and need approval before merging into master

However you can still make a change locally and push to master. It might be I'm missing something in my set up, but that's been my experience so far.

I do still agree with the standard though, its just the description might need to be altered if my experience is right.